README.md

```diff
@@ -49,7 +50,43 @@ Similarly, [calorimetric.json](https://github.com/building-envelope-data/api/blo
 
 With you web browser, you can search our [wiki](https://github.com/building-envelope-data/api/wiki), the [issues](https://github.com/building-envelope-data/api/issues) and [pull requests](https://github.com/building-envelope-data/api/pulls) and contribute to them.
 
-In order to browse the code conveniently, you should first [clone the repository](https://docs.github.com/en/free-pro-team@latest/github/creating-cloning-and-archiving-repositories/cloning-a-repository) and then use a text editor, for example [Visual Studio Code](https://code.visualstudio.com/).
+### In your browser
+
+In order to browse the code conveniently, you can click on `Code` and then `+` to start a new codespace.
+
+![Start a new codespace](docs/images/open_codespace.png)
```

Comment:
Bei mir heisst das "Create" und nicht "Start" auf GitHub.

Reply:
I have removed the image, because the description [fde4711](https://github.com/building-envelope-data/api/pull/336/commits/fde4711fff3999f6dfea68cac2c6f2fa9a2202b9) seems sufficient.